ethics/21-distraction-fr.md

```diff
@@ -0,0 +1,338 @@
+
+Et très clairement, à cette conférence, Big Tech était surreprésenté,
+qu'il s'agisse de Facebook, Google ou autres OpenAI,
+par opposition aux académiques qui n'ont généralement pas accès
+à suffisamment de supercalculateurs pour effectuer les calculs monstrueux
+nécessaires pour concevoir des algorithmes surhumains en mathématiques.
+https://www.ihp.fr/en/events/conference-flaim-formal-languages-ai-and-mathematics
+
+Qui plus est, à côté des dizaines de millions de dollars nécessaires 
+pour payer la facture électrique des supercalculateurs qui font des mathématiques,
+les salaires cumulés des chercheurs présents se chiffraient eux aussi certainement en millions,
+voire en dizaines de millions.
+Mais donc, pourquoi ? Pourquoi Big Tech investit autant dans des domaines qui, clairement,
+ne sont pas les plus utiles pour leurs produits, et donc a priori pour leurs revenus ?
+
+Pourquoi Facebook, pourtant dans le dur financièrement, 
+investit autant dans le machine learning pour les mathématiques ?
+Pourquoi Google a-t-il racheté DeepMind, 
+et les félicite de faire des algorithmes 
+pour bien jouer aux échecs, au go et aux jeux vidéos,
+ou pour résoudre le problème du repliement de protéine ?
+
+Et pourquoi OpenAI investit autant pour produire des IA "artistiques", 
+capables de générer des images spectaculaires
+ou de converser de manière bluffante ?
+Qu'est-ce qui justifie, d'un point de vue économique, le salaire de ces chercheurs ?
+
+Telles sont les questions que je n'ai cessé de poser aux participants à la conférence.
+Et, en gros, personne ne s'était même posé la question.
+Tous étaient trop distraits par la spectacularité des algorithmes développés.
+Et bien, il me semble que c'était justement ça, l'objectif économique de cette conférence.
+Il s'agissait de distraire les chercheurs des enjeux économiques et géopolitiques
+des algorithmes de machine learning,
+et de donner l'impression que l'IA, c'est cool,
+voire que freiner son progrès spectaculaire serait immoral.
+
+(alors que nombres de leurs collègues, notamment Frances Haugen et Sophie Zhang,
+appellent désespérément à "ne pas fermer les yeux"
+sur les atrocités auxquels les algorithmes de Meta ont largement contribué,
+comme le génocide de Rohingyas et du Tigray, le chaos informationnel du COVID,
+voire les émeutes du Capitol et la tentative de coup d'état par l'extrême droite en Allemagne...)
+
+Le #coolwashing a terriblement bien marché.
+
+
+## L'industrie de la distraction stratégique
+
+Il y a aujourd'hui une confusion dérangeante mais très répandue
+entre désinformation et fausse information.
+Comme ci, désinformer, c'était uniquement produire des "fake news".
+Cependant, les chercheurs qui ont choisi d'étudier les producteurs de désinformation
+constatent au contraire que l'arsenal utilisé par ces derniers transcende de loin
+la simple production de fausses informations.
+Pour dés-informer, c'est-à-dire enlever de l'information, mentir n'est pas nécessaire.
```

Comment:
Ca marche même mieux quand c'est "vrai". Ca me fait penser à une citation de la professeur Repnikova ""It's powerful, because they are not making it up," https://www.bbc.com/news/world-us-canada-52912241